gitlab/resource_gitlab_user_test.go

```diff
@@ -146,6 +147,7 @@ type testAccGitlabUserExpectedAttributes struct {
 	SkipConfirmation bool
 	External         bool
 	Note             string
+	State            string
 }
 
 func testAccCheckGitlabUserAttributes(user *gitlab.User, want *testAccGitlabUserExpectedAttributes) resource.TestCheckFunc {
```

Comment:
You'll need to update this function to check the new attribute

Reply:
Created a new one instead below to cover the blocked use case. Hope that's fine?

Reply:
Looks good!